body.tex

```diff
@@ -59,11 +59,54 @@ \subsection{Basic Real/Bogus Classifier}
+More specifically, we need to define the term ``real'' more precisely: a non-astrophysical detection, such as an artifact, is obviously not ``real''. But how about a non-varying astrophysical source?
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=.8\textwidth]{material/score-space}
+  \caption{The 2-dimensional realness space}
+  \label{fig:score-space}
+\end{figure}
+
+As illustrated in figure~\ref{fig:score-space}, the real-ness space is composed of (at least) two dimensions, which we name ``transient-ness'' and ``object-ness''.
+
+As a data-driven approach, which is being implemented as a supervised task at its core, we do not excert the definition of the problem into the method itself, but the ``truth labels'' used for training the classifier.
+\begin{tcolorbox}
+We label any suggested candidate as a \emph{positive}, if and only if it matches one of the astrophysical transients in the truth catalog, \emph{spatially} and with a high enough precision. Any other candidate is labeled as a \emph{negative}.
+\end{tcolorbox}
+
+This way, in the training phase we inject our prior knowledge about the ``real-ness'' of each object into the learning process, regardless of the difficulty of the detection of each object and without adding artificial Aleatoric uncertainty to the system. Later during the test phase (i.e. normal working mode of the pipeline), we use the pseudo-probabilities appearing before thresholding at the final layer of the network as a proxy of the score, which capture the Epistemic uncertainty of the model too.
+
+
+% There are multiple available options:
+% \begin{itemize}
+% \item simple binary classification -- the resulting score may not be quite informative
+% \item regression -- more difficult to define
+% \item multi-class classification -- difficult data annotation
+% \end{itemize}
+
+% If we go with regression... there are bogus detections, variables, ``appearing transients''
+% \begin{equation}
+%   s=\left| \frac{A_2-A_1}{A_2+A_1} \right|
+% \end{equation}
+
+
+\subsection{Drawbacks}
 \begin{itemize}
-\item has no access to the temporal~\footnote{throughout this document the term ``temporal'' is used to represent the transition from a template/reference image to a science image. It has to do, but should not be confused with the ``real'' time axis!} behaviour of the source. This is by definition outsourced to the upstream image differencing module and hence there is no ``learned intelligence'' in processing the ``transient-ness'' of the candidate.
+\item has no access to the temporal~\footnote{throughout this document the term ``temporal'' is used to represent the transition from a template/reference image to a science image. It has to do, but should not be confused with the ``real'' time axis!} behavior of the source. This is by definition outsourced to the upstream image differencing module and hence there is no ``learned intelligence'' in processing the ``transient-ness'' of the candidate.
```

Comment:
With a classifier taking three stamps, though, this isn't true: aside from false negatives (which you cover in the next bullet) the classifier has no disadvantage relative to the image differencing algorithm.

If we can avoid the "temporal" language here, again I think other readers will be less confused.

Reply:
Mostly agreed -- there is the crop-based-on-diff concept too. I think the explanation in Section 4 covers this in an elaborate fashion. I'm adding a pointer to that section too.

Reply:
Oh, do you have any suggestions for a replacement for the term "temporal"? 
Actually, I like it a lot, as it fits well with "spatial", and together they form the two main dimensions in which we seek information...

Reply:
Again, I think the fact that you can do classical RB with three stamps makes this discussion moot.